swagger.yaml

```diff
@@ -371,3 +381,28 @@ components:
           enum: ["data", "bundle_archive", "bundle_directory"]
         checksum:
           type: string
+    Edge:
+      description: |
+        A graph edge that describes a relationship between two resources
+      required:
+        - label
+        - source
+        - target
+      properties:
+        label:
+          type: string
+        source:
+          $ref: '#/components/schemas/Node'
+        target:
+          $ref: '#/components/schemas/Node'
+    Node:
+      description: A graph node that describes a resource
+      required:
+        - resource_type
+      properties:
+        id:
+          type: string
+        resource_type:
```

Comment:
I think using `type` is more consistent; elsewhere in this swagger.yaml the `id`/`type`/`namespace` either all 3 have the `resource_` prefix, or none of the three have it.